### PR TITLE
fix: ignore  RawKeyDownEvent for remove button press

### PR DESCRIPTION
**Pull request by Thioby**
This PR fixes bug presented in video:

https://github.com/danger-ahead/simple_chips_input/assets/8046733/0fceab6e-fa7b-4f5d-b21e-caf8d24f56ca

### CODE REVIEW

**Comment by danger-ahead** [comment]
Will take a look shortly.

File: lib/src/simple_chips_input.dart [hunk 1]
```diff
@@ -101,11 +104,14 @@ class SimpleChipsInput extends StatefulWidget {
   /// Callback when a chip is deleted. Returns the deleted chip content and index.
   final void Function(String, int)? onChipDeleted;
 
+  final void Function(String)? onChipAdded;
```
Comment (danger-ahead) [inline lib/src/simple_chips_input.dart:108]: Can you please add comments here, so it's easy for the devs to understand?
Adding a comment here also adds it to the API reference in pub dev

**Review by danger-ahead — CHANGES_REQUESTED** [review]
LGTM. Add some docs for the changes.

File: lib/src/simple_chips_input.dart (hunk 1 above)
Comment (Thioby) [reply lib/src/simple_chips_input.dart:108]: done

**Comment by Thioby** [comment]
Done, please release new version of lib :)

**Review by danger-ahead — APPROVED** [review]

**Pull request merged**